### ISSUE TITLE
Deviations table has more columns than indicated in the header

### ISSUE THREAD

**Issue by PatrickRWright** [SwissClinicalTrialOrganisation/secuTrialR#231]
**Describe the bug**
This is a new exception related to a table which can be present in the export called "deviations_*". Since we hardly ever use this table it has not yet come up. When this table is present an Error can be thrown. `read_secuTrial()` catches it and makes some suggestions which are not useful in this case. 

`read_secuTrial_raw()` is more specific and points out what the actual issue is:
```
Error in read.table(file = archive_con, header = TRUE, na.strings = export_options$na.strings,  : 
  duplicate 'row.names' are not allowed
```

Setting `row.names = NULL` fixes the Error but the column headers are then wrong.

The problem is that the source file has more data columns than header columns.

**To Reproduce**
Run `read_secuTrial_raw()` with an export containing a "deviations_*" file.

**Expected behavior**
Read data as usually expected.

**Additional context**
This issue was kindly reported by a colleague from Zürich.

**Comment by PatrickRWright** [comment]
This has been forwarded to iAS.

**Comment by PatrickRWright** [comment]
Should be fixed from secuTrial versions 6.1 and up.
